src/api.ts

```diff
@@ -18,3 +16,4 @@ const limiter = rateLimit({
 // } as bodyParser.Options));
 
 apiRouter.post("/upload", limiter, endpoints.upload);
+apiRouter.post("/uploadCloudinary", limiter, endpoints.uploadAndCloudinary);
```

Comment:
does this have to be a separate endpoint? I would assume "upload" endpoint can accept a "to" parameter and handle the difference internally

Reply:
Figured this would make it easier to integrate / more backward compatible.
Additionally `/upload` works for any file, but `/uploadCloudinary` only really works for picture/video files.